intake/interface/source/gui.py

```diff
@@ -154,6 +155,10 @@ def on_click_pars_widget(self, event):
         else:
             self.description.panel.remove(self.pars_editor.panel)
 
+    def on_plot_edited(self):
+        # Manually triggered from plot widget when plot is saved
+        self.description.source = self.sources
```

Comment:
Yes, essentially a "redraw" of the widget showing the YAML. I decided to revamp this to look more like a normal widget "watch" rather than passing the callback when creating the Plot.